src/index.ts

```diff
@@ -11,15 +11,19 @@ const DELIMITER_KEYWORD = 'DELIMITER'
 export interface SplitOptions {
   multipleStatements?: boolean
   retainComments?: boolean
+  includePositions?: boolean
 }
 
 interface SqlStatement {
   value: string
   supportMulti: boolean
+  start?: number
+  end?: number

```

Comment:
For simplicity, make `start` and `end` mandatory, so we can remove all those `if (includePositions)` statements in the source code.
```suggestion
  start: number
  end: number
```